### DISCUSSION TITLE
ZigStar LAN Gateway is a wired Ethernet bridge open source HW based on ESP32 that should be compatible with Zigbee2Tasmota (Z2T)

### DISCUSSION THREAD

**Discussion by Hedda** [arendst/Tasmota#13537] [Ideas]
Tip: ZigStar LAN Gateway is based on ESP32 and TI CC2652P been released as open source hardware by Grecu Radu / Darth PCB:

https://zig-star.com/projects/zigbee-gw-lan/

There are three version of it which differ in which Zigbee radio module is used (RF-STAR RF-BM-2652P2 recommended):
- RF-STAR RF-BM-2652P2 - CC2652P = +20 dBm amplification if enabled in application configuration (otherwise +5 dBm)
- Ebyte E72-2G4M20S1E - CC2652P = +20 dBm amplification if enabled in application configuration (otherwise +5 dBm)
- Ebyte E72-2G4M05S1F - CC2652RB = +5 dBm amplification

https://github.com/mercenaruss/zigstar_gateways

https://github.com/xyzroe/ZigStarGW-FW

By default the ESP32 on these are flashed with ZigStarGW FW which is a fork of ZiGate Ethernet discussed here:

https://github.com/arendst/Tasmota/discussions/12902

PS: s-hadinger mentioned there that while not tested and confirmed working, the hardware should work with Zigbee2Tasmota (Z2T) if it is based on ESP32 and uses either TI/ZNP or Silabs/EZSP based Zigbee radio module.

https://tasmota.github.io/docs/Zigbee/

**Comment by Jason2866** [comment]
Well, if you want to know if it works. Buy one and try. Sorry we can not buy every device out there.

**Reply by Hedda** [reply]
It is just a hardware tip. I am not saying that it has been tested and confirmed working as of yet.

**Reply by mercenaruss** [reply]
@Hedda i will run a test,for sure is working,is the same TI CC2652P IC inside. :)

**Reply by Hedda** [reply]
@mercenaruss as the maker of ZigStar might want to consider donating one GW each to s-hadinger, Jason2866, ascillato, and arendst

https://github.com/arendst/Tasmota/graphs/contributors
